packages/odyssey-react-labs/src/datePickerTheme.tsx

```diff
@@ -76,20 +77,60 @@ const todayDotStyles: StateStyles = {
   }),
 };
 
+const yearStyles: StateStyles = {
+  default: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.primary,
+  }),
+  disabled: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.disabled,
+  }),
+  hover: ({ theme }) => ({
+    backgroundColor: theme.palette.grey[100],
+  }),
+  hoverSelected: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.lighter,
+    color: theme.palette.primary.main,
+  }),
+  outsideOfMonth: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.secondary,
+  }),
+  selected: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.primary.main,
+    fontWeight: theme.typography.fontWeightBold,
+  }),
+};
+
+const yearCheckStyles: StateStyles = {
+  default: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    content: '""',
+    height: `${(16 / 16) * (16 / 14)}rem`,
+    maskImage:
+      "url(\"data:image/svg+xml,%3Csvg viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%0A%3E%3Cpath fillRule='evenodd' clipRule='evenodd' d='M14.3536 4.35355L6.35355 12.3536C6.15829 12.5488 5.84171 12.5488 5.64645 12.3536L1.64645 8.35355L2.35355 7.64645L6 11.2929L13.6464 3.64645L14.3536 4.35355Z' fill='currentColor' /%3E%3C/svg%3E%0A\")",
+    maskPosition: "50% 50%",
+    maskRepeat: "no-repeat",
+    position: "absolute",
+    right: theme.spacing(4),
+    width: `${(16 / 16) * (16 / 14)}rem`,
+  }),
+};
+
 export const datePickerTheme: ThemeOptions = {
   components: {
     MuiCalendarPicker: {
       styleOverrides: {
         root: ({ theme }) => ({
+          borderColor: theme.palette.divider,
+          borderStyle: "solid",
+          borderWidth: "1px",
+          borderRadius: `${(4 / 16) * (16 / 14)}rem`,
```

Comment:
Typically, design doesn't want these to scale up, so you can use `theme.mixins.borderRadius` here. If we decide to make them scaling, we can change it at the token level.

`theme.mixins.borderStyle` and `theme.mixins.borderWidth` are available for the other properties as well

Reply:
One of them is 4px, the other is 6px. This is will make the design not match. Do we just want them all at 4px instead?

Reply:
@taylorridgway-okta @josesolano-okta should we have a token for the `6px` border radii?

In either case, we shouldn't use the calculation inline here.